backend/src/kitconcept/intranet/upgrades/v20250401001/__init__.py

```diff
@@ -0,0 +1,12 @@
+from kitconcept.intranet import logger
+from plone import api
+from Products.CMFPlone.MigrationTool import MigrationTool
+from Products.GenericSetup.tool import SetupTool
+
+
+def upgrade_plone_611(setup_tool: SetupTool):
+    """Upgrade Products.CMFPlone to version 6.1.1."""
+    request = api.env.getRequest()
+    migration_tool: MigrationTool = api.portal.get_tool("portal_migration")
+    result = migration_tool.upgrade(request, dry_run=False, swallow_errors=True)
+    logger.info(f"Upgrade to Plone 6.1.1. {result}")
```

Comment:
There's nothing version-specific here. This can be a generic function that we reuse as an upgrade step whenever we upgrade Plone.